### ISSUE TITLE
Browser compatibility

### ISSUE THREAD

**Issue by Weestdudochnicht** [madcowpp/dcDragAndDrop#1]
Your Plugin doesnt work for Firefox. Please fix it.

**Comment by madcowpp** [comment]
That is not true, the directives work in Firefox, it's the demo which does not work. This is due to the use of 'event.srcElement' in the handler functions (which Firefox does not support, see [here](https://developer.mozilla.org/en/docs/Web/API/Event/srcElement)). You are welcome to provide a demo which works in Firefox as well, which can be done by replacing 'e.srcElement' by 'e.target' in demo.js. The demo also does not work properly in Chrome, because Chrome does not allow access to dataTransfer.getData properties in dragOver events.

**Comment by Weestdudochnicht** [comment]
sorry. i missed to use the setData, what is necessary for firefox...

 event.originalEvent.dataTransfer.setDragImage(element, 10, 10);
 event.originalEvent.dataTransfer.setData('text', null);

**Comment by madcowpp** [comment]
Since the demo indeed does not work with firefox, I'm changing the issue. I'm not gonna fix that, because I'm not working with Angular anymore. If anyone is interested I wrote how to fix it already in my first comment.
